src/data_handler/data_handler.cpp

```diff
@@ -891,18 +882,155 @@ int get_psd_welch (double *data, int data_len, int nfft, int overlap, int sampli
     return (int)BrainFlowExitCodes::STATUS_OK;
 }
 
-int get_log_psd_welch (double *data, int data_len, int nfft, int overlap, int sampling_rate,
-    int window_function, double *output_ampl, double *output_freq)
+int get_avg_band_powers (double *raw_data, int rows, int cols, int sampling_rate, int aply_filters,
+    double *avg_band_powers, double *stddev_band_powers)
```

Comment:
Not a big deal, but I would like to see a big function like this broken up into logical chunks that are easier to read and maintain.

For example there's a parallel for loop in here. Could that be it's own function?